theme_vehicle/views/customizations.xml

```diff
@@ -1,171 +1,365 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <!-- ======== BANNER ======== -->
+    <template id="s_banner" inherit_id="website.s_banner">
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" remove="s_parallax_is_fixed" separator=" "/>
```

Comment:
To check: if you only remove the s_parallax_is_fixed class, is not there still something like scroll-background-ratio + parallax class?